Source/WebKit/WPEPlatform/wpe/WPEView.h

```diff
@@ -138,6 +138,7 @@ WPE_API guint                   wpe_view_compute_press_count           (WPEView
                                                                         gdouble             y,
                                                                         guint               button,
                                                                         guint32             time);
+WPE_API void                    wpe_view_clear_press_count             (WPEView            *view);
```

Comment:
It seems counter-intuitive that an external entity has to tell the view that it has to clear something, because of a navigation/load action. I guess there is never a situation where we want to preserve that press count between such events, no?

Reply:
IIUC, `WPEView` is in charge of lower level operations like delivering events to the `WebKitWebView` and forwarding buffers received from it. Something like a "shell" around `WebKitWebView`. So, `WPEView` would not have knowledge of web operations like loading, navigation, etc.

Maybe instead of the `EventSender` directly calling this new function, should we make `WebKitWebView` call it when those cases happen, like loading a new page? 

Another option, if we want to avoid creating a new function, would be creating something like a `WPEEventType::WPE_EVENT_RESET`, that, when sent through `wpe_view_event(event)`, tells the `WPEView` to reset its event state tracking.

Reply:
That's not GTK4 but GTK3 and it's not used by layout tests. EventSenderProxy always passes its computed click count value to the synthesized event that it's used directly instead of the value computed by the web view in the case of GTK3 or the value computed by GTK in the case of GTK4. The reason why this fails in WPE is becase we use  WPEView to compute the value, which is the same for tests run by the same WTR. All other ports compute the click count in EventSenderProxy. A new EventSenderProxy is created for test even in the same WTR so click count is always reset for every test. So, I think we should follow all other ports and compute the click counter in EventSenderProxy instead of using WPEView.